arm/Microsoft.NetApp/netAppAccounts/capacityPools/deploy.bicep

```diff
@@ -46,46 +46,52 @@ module pid_cuaId '.bicep/nested_cuaId.bicep' = if (!empty(cuaId)) {
   params: {}
 }
 
-resource capacityPool 'Microsoft.NetApp/netAppAccounts/capacityPools@2021-06-01' = {
-  name: '${netAppAccountName}/${name}'
-  location: location
-  tags: tags
-  properties: {
-    serviceLevel: serviceLevel
-    size: size
-    qosType: qosType
-    coolAccess: coolAccess
+resource netAppAccount 'Microsoft.NetApp/netAppAccounts@2021-04-01' existing = {
+  name: netAppAccountName
+  resource capacityPool 'capacityPools@2021-06-01' = {
```

Comment:
This nested approach is still under discussion (ref. pinned discussion in the internal channel). Not sure if worthy to refactor the module at this level already. @MrMCake thoughts? I sense we'll end up keeping the main resource at top level.

Reply:
I'd personally prefer the nesting, but it's as we said in the channel - not agreed upon. For now we have different approaches merged. I.e. in storage account we have the nesting, in automation account the nesting aside from the top level resource, and most if not all others use neither but instead string-concat. 

Hence I'd not have a strong opinion about it and would be fine either way.

However - we we keep it as is @Msanzdelrio then please update line 87 & reference `netApAccount.name`